### PR TITLE
New Tutorial - Install and Configure Plex Media Server on CentOS 8

### PR DESCRIPTION
**Pull request by mayankdebnath**
It's 5am, Been working on this since all night, Good Morning from India, Do let me know if any changes are required! Thanks.

### CODE REVIEW

File: tutorials/install-and-configure-plex-media-server-on-centos-8/01.en.md [hunk 1]
```diff
@@ -0,0 +1,262 @@
...
+
+> If you already have a local Plex Media Server installed, You need to stop it & then verify by opening `http://127.0.0.1:32400/web` it should not give any output before doing any of the steps mentioned below.
+
+### Step 3.1 - SSH Tunnel
+
+To access server's localhost on our localhost we need to setup SSH tunneling by running one simple command.
+If you're on windows run it on CMD or if you're on linux then use your bash shell.
+
+```bash
+ssh -L 127.0.0.1:32400:127.0.0.1:32400 root@<server_public_ip_here>
```
Comment (n-se) [inline tutorials/install-and-configure-plex-media-server-on-centos-8/01.en.md]: Please use the example IP for the server mentioned in https://github.com/hetzneronline/community-content/blob/master/tutorial-template.md#introduction

File: tutorials/install-and-configure-plex-media-server-on-centos-8/01.en.md [hunk 2]
```diff
@@ -0,0 +1,262 @@
...
+
+![Server Setup Page](images/03-server-setup-page.PNG)
+
+> Make sure the "Allow me access my media outside my home" option remains enabled.
+
+For the remaining 2 tabs, Just hit "Next" because those can be configured later on.
+
+#### Step 3.2.3 - Accessing Plex Media Server over Internet
+
+Now that you've completed the setup prompt, You can open the Plex Media Server page by opening `http://<server_public_ip_here>/web`.
```
Comment (n-se) [inline tutorials/install-and-configure-plex-media-server-on-centos-8/01.en.md]: same here with the IP as above

**Comment by mayankdebnath** [comment]
Thanks for the appreciation Noah :), I am glad you liked it, I have been using Hetzner servers for a long time & I want to be part of the community. 

I have done both the changes, I could've added the reverse proxy part but it's kind of long, I am thinking to make 2 new tutorials in the weekend:

1. Install and configure Plex Media Server on Ubuntu 19/Debian 10
2. Setup Nginx as Reverse Proxy for Plex Media Server

What do you say?
Thank you!

**Comment by n-se** [comment]
@mayankdebnath we would be very happy about the two additional tutorials :)
Good idea to create an extra tutorial for the plex reverse proxy since there is more needed than just a simple `proxy_pass`.

**Comment by mayankdebnath** [comment]
Thanks a lot, @n-se. ;)

**Comment by mayankdebnath** [comment]
How do I start writing the new ones because if I push another commit it'll be added to this pull request, Do I make a new branch and then send pull request or something else?

**Comment by n-se** [comment]
This is currently kinda a problem because these commits are on your master branch.
GitHub cannot handle this, if you create a new branch based on master the commits from this tutorial will be in the new one too.
You would need to rebase the new branch on upstream master, this needs git cli knowledge. Maybe you just write the tutorial external and after this tutorial gets merged (I will ping the community manager to do this as soon as possible), you delete and fork again and then create a new branch based on master.

You can "test" or write your markdown for the new tutorials with https://demo.codimd.org/

**Comment by mayankdebnath** [comment]
It's fine, I'll wait for it to get merged.

**Comment by mayankdebnath** [comment]
Hey @leojakeson,

Thanks a lot for merging my pull request. 

I see you have done a few changes, I have analyzed the mistakes & it'll help me to become better at this. 

How much time does it usually take for a merge to affect, Can't find this tutorial on https://community.hetzner.com/tutorials/

Thanks again :)

**Comment by n-se** [comment]
The CI job must be approved internally, after that is done it gets published on the community page. Should probably happen today :)

**Pull request merged**